themes/default/content/blog/iac-recommended-practices-wrapping-up/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Iac Recommended Practices: Wrapping Up"
+date: 2024-03-11
+draft: false
+meta_desc: This wraps up the series on IaC recommended practices, summarizing the previous posts and pointing out areas of future growth for the Zephyr team.
+meta_image: meta.png
+authors:
+    - aaron-kao
+    - christian-nunciato
+    - scott-lowe
+tags:
+    - best-practices
+    - zephyr
+---
+
+Welcome to the final post in our series of articles focused on infrastructure as code (IaC) recommended practices. In this post, we'll wrap up our recommendations for IaC with Pulumi, summarizing the recommendations from previous posts as well as highlighting some areas of potential future growth for the team at Zephyr Archaeotech Emporium---the fictional company at the center of the story throughout this series.<!--more-->
+
+For ease of navigation, here are links to all the blog posts in the series (any entries listed below that aren't linked haven't yet been published; this list will get updated as new posts go live):
+
+* [IaC Recommended Practices: Code Organization and Stacks](/blog/iac-recommended-practices-code-organization-and-stacks/)
+* [IaC Recommended Practices: Developer Stacks and Git Branches](/blog/iac-recommended-practices-developer-stacks-git-branches/)
+* [IaC Recommended Practices: Structuring Pulumi Projects](/blog/iac-recommended-practices-structuring-pulumi-projects/)
+* [IaC Recommended Practices: Using Stack References](/blog/iac-recommended-practices-using-stack-references/)
+* [IaC Recommended Practices: RBAC and Security](/blog/iac-recommended-practices-rbac-and-security/)
+* [IaC Recommended Practices: Using Automation API](/blog/iac-recommended-practices-using-automation-api/)
+* **IaC Recommended Practices: Wrapping Up** (this post)
+
+## Recapping infrastructure as code with Pulumi at Zephyr
+
+As this series has progressed, we've been showing you how Zephyr's use of Pulumi has changed in response to the organization, the team, and the application has changed. In case you haven't been following along---or in case you need a refresher---here's a brief summary of what's happened in the series:
```

Comment:
```suggestion
As this series has progressed, we've been showing you how Zephyr's use of Pulumi has changed in response to how the organization, the team, and the application has changed. In case you haven't been following along---or in case you need a refresher---here's a brief summary of what's happened in the series:
```

Reply:
Fixed, thanks!